.gitignore

```diff
@@ -5,3 +5,4 @@ build
 .lock*
 *.swp
 *.vscode*
+gae/secrets.py
```

Comment:
you can remove this, it won't commit `secret.py`

also it should be `secret` not `secrets`